src/cscore_cpp.cpp

```diff
@@ -619,7 +619,7 @@ std::string GetHostname() {
   name[255] = '\0';  // Per POSIX, may not be null terminated if too long
   return name;
 #else
-  return "";  // TODO
+  return "";                          // TODO
```

Comment:
Wow. clang-format's comment alignment broke horribly there. 4.0.1 breaks in the same way. I can't figure out why it happened, so I'll disable comment alignment.